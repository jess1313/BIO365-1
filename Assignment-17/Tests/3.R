suppressPackageStartupMessages(suppressWarnings(suppressMessages(source("3.R"))))

set.seed(0)

print(getPWithin(rnorm(1000), -3, 1.5))
print(getPWithin(rnorm(1000), 1.0, 1.1))
print(getPWithin(rnorm(1000), -2, 10))
