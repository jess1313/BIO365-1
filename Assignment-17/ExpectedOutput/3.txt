[1] 0.9342069
[1] 0.0224791
[1] 0.9848853
[1] 0
